furusiya/model/components/ai/base.py

```diff
@@ -0,0 +1,12 @@
+from model.components.base import Component
+
+
+class AI(Component):
```

Comment:
I don't like this (the filename and class name don't match`. Please call this `AIBaseClass` or (unless I'm mixing C#/Java and Python concepts) `AbstractAI`.

And is it `AI` or `Ai`?

Reply:
`AI`. The names don't necessarily need to match.

I say it's clear enough as-is, but if we must, I guess we can change it.